client/components/theme-collection/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button } from '@wordpress/components';
+import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
+import { ReactElement, useEffect, useRef, PropsWithChildren, useState } from 'react';
+import './style.scss';
+import { Swiper as SwiperType } from 'swiper/types';
+
+interface ThemeCollectionProps {
+	collectionSlug: string;
+	heading: string;
+	subheading: ReactElement;
+}
+
+export default function ThemeCollection( {
+	collectionSlug,
+	heading,
+	subheading,
```

Comment:
How do you feel about renaming these into `title` and `description`? IMHO it's more clear at first glance.

Reply:
sure :)